opt/scopeserver/dataserver/utils/loom.py

```diff
@@ -704,14 +705,18 @@ def has_regulons_AUC(self) -> bool:
 
     def get_regulons_AUC(self, regulon_type: str = None):
         loom = self.loom_connection
-        if regulon_type == "motif":
+        if "MotifRegulonsAUC" in self.loom_connection.ca.keys() and regulon_type == "motif":
```

Comment:
Can you update the variables here from `L` to something i) Lower case ii) More descriptive?